test/api/buffer/api_test.go

```diff
@@ -135,9 +136,18 @@ func RunE2ETest(t *testing.T, testDir, workingDir string, binary string) {
 		assert.NoError(t, err)
 	}
 
-	// Assert
+	// Request
 	requestsOk := RunRequests(t, envProvider, testDirFs, workingDirFs, apiUrl, etcdClient)
 
+	// Shutdown API server
+	_ = cmd.Process.Signal(syscall.SIGTERM)
+	select {
+	case <-time.After(10 * time.Second):
+		t.Fatalf("timeout while waiting for server shutdown")
+	case <-cmdWaitCh:
+		// continue
+	}
+
```

Comment:
Predtym ako dumpneme `etcd kvs`, musime vypnut server.
Je to pre `statistics`, ktore sa nahravaju kazdu sekundu a pri vypnuti serveru.
Vypnutim servera zabezpecime, ze su vsetky statistiky synchronizovane do etcd.